packages/shell-api/src/decorators.ts

```diff
@@ -350,12 +351,14 @@ function getShellInstanceState(apiObject: any): ShellInstanceState | undefined {
  * as needed.
  */
 export interface ShellCommandAutocompleteParameters {
-  getCollectionCompletionsForCurrentDb: (
-    collName: string
-  ) => string[] | Promise<string[]>;
   getDatabaseCompletions: (dbName: string) => string[] | Promise<string[]>;
 }
 
+export type NewShellCommandAutocompleteParameters = Pick<
```

Comment:
Don't know how obvious it is from the diff, but this is just new types everywhere ShellCommand* was with that now the legacy type. We'll have to just clean this up when we remove the old autocompleter.

Again: open to suggestions for how to rather do this.